snap/local/start-mysqld.sh

```diff
@@ -7,9 +7,10 @@ if [ ! -f "${CONFIG_DIR}/initialized" ]; then
         echo "copying default config to ${CONFIG_DIR}"
         cp -r $SNAP/etc/mysql $SNAP_COMMON
         sed -i "s:/var/run/mysqld/mysqld.sock:$SNAP_COMMON/mysqld.sock:g" $SNAP_COMMON/mysql/mysql.conf.d/mysqld.cnf
+        sed -i "s: mysql: root:g" $SNAP_COMMON/mysql/mysql.conf.d/mysqld.cnf
         sed -i "s:# sock:sock:g" $SNAP_COMMON/mysql/mysql.conf.d/mysqld.cnf
         sed -i "s:/var/log/mysql/error.log:$SNAP_COMMON/error.log:g" $SNAP_COMMON/mysql/mysql.conf.d/mysqld.cnf
         touch $CONFIG_DIR/initialized
 fi
 
-$SNAP/usr/sbin/mysqld --defaults-file=$SNAP_COMMON/mysql/mysql.conf.d/mysqld.cnf
+$SNAP/usr/sbin/mysqld --defaults-file=$SNAP_COMMON/mysql/mysql.conf.d/mysqld.cnf --user=root
```

Comment:
Can you please post details in Git commit.
The current one "Fix cli params for mysqld" doesn't match the content of the commit "Switch from user mysql to root"
IMHO, the good commit message should answer "What? Why? How?". Can you please add "Why?" at least?

Reply:
I have updated the PR description.